### PR TITLE
feat(users): added generic unvalidated users page/actions

### LINKED ISSUE #4365: user validation by email: handling of unapproved user entities after uservalidationbyemail is disabled (Trac #4365)
_Original ticket http://trac.elgg.org/ticket/4365 on 42112849-12-10 by trac user ismayil.khayredinov, assigned to unknown._

Elgg version: 1.8.3

Just noticed that if you have unvalidated users and uservalidationbyemail is disabled there is no way to validate the users. Perhaps there should be some way of processing unapproved users - either deleting them or automatically approving.

### LINKED ISSUE #4561: Basic Uservalidation (Trac #4561)
_Original ticket http://trac.elgg.org/ticket/4561 on 42424863-08-07 by trac user webgalli, assigned to unknown._

Elgg version: 1.8.5

Have a better user validation plugin so that we can decide on which option to select in the admin panel. Validation by admin or by Email.

See http://community.elgg.org/pg/plugins/project/875414/developer/webgalli/user-validation-by-admin-for-18x

### PR DESCRIPTION
**Pull request by jeabakker**
BREAKING CHANGE: The unvalidated users page and some actions have been
moved to core and are no longer present in the uservalidationbyemail
plugin.

fixes #4561
fixes #4365

ToDo
- [x] docs
- [x] ~~improved flow of validating user~~ (going to do this in a separate PR)

### CODE REVIEW

**Comment by jeabakker** [comment]
going to do the new method of user validation in an other PR, as that requires a bit more work and this will help non the less.

also rebased

**Review by jdalsem — CHANGES_REQUESTED** [review]

File: actions/admin/user/validate.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<?php
```
[the remark this answers is not in the document]
Comment (jeabakker) [reply actions/admin/user/validate.php]: a bit more feedback

**Review by jdalsem — APPROVED** [review]

**Comment by jdalsem** [comment]
rebase and this is ready to merge

**Comment by jeabakker** [comment]
rebased

**Pull request merged**